src/pages/rekey-reason/rekey-reason.scss

```diff
@@ -21,11 +84,32 @@ page-rekey-reason {
     font-size: 3.4rem;
   }
 
+  .mes-return-button {
+    @extend .mes-primary-button;
+    margin-bottom: 36px;
+    width: 40%;
+    background: map-get($colors-gds, gds-grey-2);
+    box-shadow: inset 0 -2px 0 0 map-get($colors-gds, gds-grey-1);
+    font-size: 20px;
+    color: black;
+    &.activated {
+      background: map-get($colors-gds, gds-grey-1);
+    }
+    h3 {
+      color: map-get($colors-gds, gds-black);
+    }
+    margin-top: 9px;
+    height: 50px;
```

Comment:
This will need to be `min-height` so that the button can grow with larger text sizes